config/manager/manager.yaml

```diff
@@ -34,7 +34,7 @@ spec:
       containers:
       - command:
         - /manager
-        image: controller:latest
+        image: $(codeflare_operator_controller_image)
```

Comment:
Shouldn't it be `codeflare-operator-controller-image` instead of `codeflare_operator_controller_image`?

Reply:
The `codeflare_operator_controller_image` is just a variable for the image in the params file.
We specify the replacement for `$(codeflare_operator_controller_image)` in the [kustomization](https://github.com/project-codeflare/codeflare-operator/pull/398/files#diff-14effe9ea1a3ddb6e419e627a3501a6c42aa2dcbc76803923e9aff2438e6ee34R15) file